### ISSUE TITLE
Wrong unit for rlink quality

### ISSUE THREAD

**Issue by ScratMan** [Minims/SomfyProtect2MQTT#75]
Hello,

After updating HA to 2023.1.1 and with SomfyProtect2MQTT release 0.2.5, I get incorrect unit warning reports for my intellitag sensors in HA : 

```
Logger: homeassistant.components.sensor
Source: components/sensor/__init__.py:980
Integration: Capteur ([documentation](https://www.home-assistant.io/integrations/sensor),
[issues](https://github.com/home-assistant/home-assistant/issues?q=is%3Aissue+is%3Aopen+label%3A%22integration%3A+sensor%22))
First occurred: 20:06:24 (22 occurrences)
Last logged: 20:07:15

  Entity sensor.baie_salon_so_droite_rlink_quality_percent (<class 'homeassistant.components.mqtt.sensor.MqttSensor'>)
  is using native unit of measurement '%' which is not a valid unit for the device class ('signal_strength')
  it is using; Please update your configuration if your entity is manually configured, otherwise create a bug report
  at https://github.com/home-assistant/core/issues?q=is%3Aopen+is%3Aissue+label%3A%22integration%3A+mqtt%22
```

I don't know why it appeared after update, but it seems some other integrations suffered from same warning:
https://github.com/home-assistant/core/pull/84376

**Comment by Minims** [comment]
Hello,

Will be fixed in 0.2.6

https://github.com/Minims/SomfyProtect2MQTT/commit/0c265c83f966a5a189435cc9bbccf3d4a06cd293

**Comment by Minims** [comment]
0.2.6 published, Please update
